src/ifs/wl_seat/event_handling.rs

```diff
@@ -278,6 +292,7 @@ impl WlSeatGlobal {
             InputEvent::TabletPadModeSwitch { .. } => {}
             InputEvent::TabletPadRing { .. } => {}
             InputEvent::TabletPadStrip { .. } => {}
+            InputEvent::Touch { .. } => {}
```

Comment:
This should call `self.cursor_group.deactivate`.

Reply:
Should it be called for all Touch events? I think at least a touch cancel shouldn't.

Reply:
Touch down and motion should be sufficient.